ext/zip/config.m4

```diff
@@ -4,36 +4,21 @@ PHP_ARG_WITH([zip],
     [Include Zip read/write support])])
 
 if test "$PHP_ZIP" != "no"; then
-  PKG_CHECK_MODULES([LIBZIP], [libzip >= 0.11])
-  LIBZIP_LIBDIR=`$PKG_CONFIG --variable=libdir libzip`
+  PKG_CHECK_MODULES([LIBZIP], [libzip >= 1.0.1])
 
-  dnl Could not think of a simple way to check libzip for overwrite support
-  PHP_CHECK_LIBRARY(zip, zip_open,
-  [
-    PHP_ADD_LIBRARY_WITH_PATH(zip, $LIBZIP_LIBDIR, ZIP_SHARED_LIBADD)
-    AC_DEFINE(HAVE_LIBZIP,1,[ ])
-  ], [
-    AC_MSG_ERROR(could not find usable libzip)
-  ], [
-    -L$LIBZIP_LIBDIR
-  ])
+  PHP_EVAL_INCLINE($LIBZIP_CFLAGS)
+  PHP_EVAL_LIBLINE($LIBZIP_LIBS, ZIP_SHARED_LIBADD)
```

Comment:
I agree with adding these 2 lines.

Reply:
BTW, if LIBZIP_CFLAGS is managed by PHP_EVAL_INCLINE, can be removed from PHP_NEW_EXTENSION